### PR TITLE
Automatic tests

### PR DESCRIPTION
**Pull request by lilachmaliniak**
merge PLAT-5838 live consistency tests

### CODE REVIEW

File: common/config/configMapping.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (kalturaguy) [inline common/config/configMapping.json]: please remove

File: common/config/config.json.template [hunk 2]
```diff
@@ -23,8 +23,30 @@
   "logFileName" : "~/dvr/kLiveController.log",
   "logTimeZoneOffset" : 300,
   "monitorPort": 4390,
+  "preserveOriginalHLS": false,
+  "recordingDurationMinutes": 5,
```
Comment (kalturaguy) [inline common/config/config.json.template]: rename and move to simualteStreams

File: common/config/config.json.template [hunk 3]
```diff
@@ -23,8 +23,30 @@
...
+  "overrideOldRecordedHLS": true,
```
Comment (kalturaguy) [inline common/config/config.json.template]: move preserveOriginalHLS into new section

preserveOriginalHLS: {
  "enable": True,
  "createFolderPerSession": True,
  "runDurationInMinutes": 5
}

File: lib/NetworkClient.js [hunk 4]
```diff
@@ -19,7 +19,11 @@ function readRequest(requestData)
             return;
         }
         if (!error && response.statusCode === 200) {
-            d.resolve(body);
+            if (!requestData.method || requestData.method != 'HEAD') {
+                d.resolve(body);
```
Comment (kalturaguy) [inline lib/NetworkClient.js]: d.resolve({body,response.headers});

File: lib/Adapters/TestAdapter.js [hunk 5]
```diff
@@ -8,15 +8,33 @@ var util=require('util');
...
+var overrideOldRecordedHLS = config.get('overrideOldRecordedHLS');
 
 function TestAdapter() {
     BaseAdapter.call(this);
+    this.setRecordingTimer();
 }
 
 util.inherits(TestAdapter,BaseAdapter);
 
+TestAdapter.prototype.setRecordingTimer = function() {
```
Comment (kalturaguy) [inline lib/Adapters/TestAdapter.js]: please move to controller

File: lib/Adapters/BaseAdapter.js [hunk 6]
```diff
@@ -1,11 +1,27 @@
...
 }
 
+util.inherits(BaseAdapter, events.EventEmitter);
+
+
 BaseAdapter.prototype.getLiveEntries = function() {};
+BaseAdapter.prototype.setControllerWrapper = function(controllerWrapper) {
+    this.controller = controllerWrapper;
+};
+BaseAdapter.prototype.gracefullyExit = function(error) {
```
Comment (kalturaguy) [inline lib/Adapters/BaseAdapter.js]: please move

File: common/config/configMapping.json (hunk 1 above)
Comment (lilachmaliniak) [inline common/config/configMapping.json]: done

File: common/config/config.json.template (hunk 2 above)
Comment (lilachmaliniak) [inline common/config/config.json.template]: ok

File: common/config/config.json.template (hunk 3 above)
Comment (lilachmaliniak) [inline common/config/config.json.template]: done

File: lib/NetworkClient.js (hunk 4 above)
Comment (lilachmaliniak) [inline lib/NetworkClient.js]: The returned promise value now contains: {body: body, headers: response.headers}.

File: lib/Adapters/TestAdapter.js (hunk 5 above)
Comment (lilachmaliniak) [inline lib/Adapters/TestAdapter.js]: changed to setRunEndTimer.

File: config.json [hunk 7]
```diff
@@ -0,0 +1 @@
...:0.01,"enable":true,"min_fps":5,"max_fps":30,"interleaving_threashold":5,"max_pts_diff":20,"chunks_to_inspect":1,"save_frame_info":false,"hashFrames":false,"ts_variance_reset":30,"verify_size":true,"pts_progress_threshold":0.2,"max_kfi":2.2,"save":false,"ffprobe_location":"ffprobe","min_kfi":1.8},"threads_to_use":10,"KalturaService":{"cdnUrl":"http:\/\/kalseglive-a.akamaihd.net:80\/dc-{0}\/m\/{1}\/kLive\/smil:{2}_all.smil\/playlist.m3u8","partnerId":-5,"adminSecret":"a9685a1598fab3f348f95ba23a36096c","serverAddress":["http:\/\/pa-www.kaltura.com","http:\/\/ny-www.kaltura.com"]},"WowzaExtractor":{"disconnection_threshold":600,"password":"kal4tura","enable":false,"retrieve_current_stats":true,"user":"kaltura"},"max_entries":1000,"bypass_CDN":{"enable":false},"retrive_entries_from_kaltura_api":true,"analyzeHlsInterval":15,"output":{"history_time":600,"history_folder":".\/history\/","reports_folder":".\/reports\/","content_folder":".\/reports\/content\/","content_duration":"60","reports_window_length":10,"save_reports":true},"valid_max_ts_duration":20,"static_entries":["http:\/\/kalseglive-a.akamaihd.net\/dc-0\/m\/pa-live-publish2\/kLive\/smil:1_oorxcge2_all.smil\/35\/chunklist.m3u8"]}
```
Comment (kalturaguy) [inline config.json]: same comment as before

File: endListPending.json [hunk 8]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (kalturaguy) [inline endListPending.json]: same comment as before

File: config.json (hunk 7 above)
Comment (lilachmaliniak) [inline config.json]: this was automatically generated based on repository. can remove it.

File: endListPending.json (hunk 8 above)
Comment (lilachmaliniak) [inline endListPending.json]: removed from branch.

**Pull request closed without merging**